src/test/unit/app/controller/courts/OpeningTimesController.ts

```diff
@@ -135,7 +152,7 @@ describe('OpeningTimesController', () => {
       'opening_times': postedOpeningTimes,
       openingTimeTypes: expectedSelectItems,
       updated: false,
-      errorMsg: 'A problem occurred when saving the opening times.'
+      errors: [{text: 'A problem occurred when saving the opening times.'}]
```

Comment:
This error message exists as a private field in the controller class. I wonder if we should make that public and just refer to the variable here to avoid the test failing if we ever want to change the text.

Reply:
I have made the test using the error string from the controller class